tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -38,21 +39,71 @@ def main() -> int:
     # Process initialization/setup must be done early. NOTE: do NOT log before this line!
     process_init(build_args)
 
-    # Return process exit code (or raise, which exits with a code of `1`)
-    return do_build(build_args, skip_completed_steps=cli_args.test_resume)
+    command = os.getenv("CENSUS_BUILD_COMMAND")
+    if command == "build":
+        return do_build(build_args)
+    elif command == "replicate":
+        return do_replicate(build_args)
+    elif command == "release":
+        return do_release(build_args)
+    elif command == "sync-release":
+        return do_sync_release(build_args)
+    elif command == "pass":
+        return 0  # Used for testing
+    else:
```

Comment:
This else branch (full build) will execute upon any unrecognized CENSUS_BUILD_COMMAND, including an unset env variable.  I'd rather error check for sanity - i.e., only do the full build when it is unset, and throw an error if it is set to an unknown value.

In other words, I'd like better error checking

Reply:
Totally agree. The full build should be performed only if `CENSUS_BUILD_COMMAND` is explicitly undefined.